### DISCUSSION TITLE
DNS amplification prevention

### DISCUSSION THREAD

**Discussion by Disqu3-mirOir-qt** [AdguardTeam/AdGuardHome#2727] [Q&A]
it,s possible added option for rejected / drop whois by country 

example allowed only Whois FR and rejected others RU BZ PT ........

because when ppl use adh for ddos if ban ip good but ip dynamic problem 

thank you

**Comment by ameshkov** [comment]
Tbh, I don't like the geo-blocking approach, I think we can come up with something smarter than that.

I've marked the issue as "question" and "help wanted". If anyone has any ideas of how to handle DNS amplification better than the current way (20rps rate limit by default), please comment here.

**Comment by Eyeborgs** [comment]
I suggest #805

**Comment by ameshkov** [comment]
#805 is reasonable, but I'd like to avoid adding new configuration settings.

**Comment by Eyeborgs** [comment]
@ameshkov Then I don't know what to offer
